PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/views/webview/mraid/BaseJSInterface.java

```diff
@@ -171,9 +171,8 @@ public String getDefaultPosition() {
     public String getCurrentPosition() {
         JSONObject position = new JSONObject();
         Rect rect = new Rect();
-
-        adBaseView.getGlobalVisibleRect(rect);
-
+        Handler mainHandler = new Handler(Looper.getMainLooper());
+        mainHandler.post(() -> adBaseView.getGlobalVisibleRect(rect));
```

Comment:
The quick search shows that there are more than 20 usages of the `adBaseView` object in the `BaseJSInterface` class. 

<img width="1209" alt="image" src="https://github.com/prebid/prebid-mobile-android/assets/21169695/3029901d-44a6-4d6c-99d2-b0137e35dfa7">

I'm afraid any of them can lead to a similar crash.

I'd suggest really, hard to reproduce the current crash first, then fix it, then try to reproduce with other usages of the `adBaseView` before the release. It will help to understand the curcomstanses and prevent other crashes, potentially in the same ads. 

If we can't reproduce it, of course, we need to release the fix based on the assumptions, but it is better to cover as much potential cases as possible. Otherwice we will get the same crash statistic but with another method, for example: ` adBaseView.getGlobalVisibleRect(rect)`

Reply:
It looks like the Handler runs the task asynchronously. However, we need the data synchronously to process it in the next step.

<img width="1067" alt="image" src="https://github.com/prebid/prebid-mobile-android/assets/21169695/6df2a9ae-926c-4e62-ae00-5613f7450115">